src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -32,6 +32,10 @@
                 "title": "Sequential Test Runs",
                 "description": "Activate to first run structural and then behavior test. This feature can help students to better concentrate on the immediate challenge at hand. If activated, you will be provided with more information in a readme file within the test repository. ATTENTION: Can not be changed after creation!"
             },
+            "automaticSubmissionRunAfterDueDate": {
+                "title": "Automatic Submission Run After Due Date",
+                "description": "Activate this option to run an automated submission run 1 hour after the due date has passed. The created results from the students' submission will be rated for this submission run even though the due date has passed."
```

Comment:
Same as above. I would also note that this is especially helpful when test cases are marked to run `After Due Date`. Why do you say 1 hour after the due date? I would say this is configurable and can be the same time as the due date.

Reply:
This was a misunderstanding from my side. I updated the PR and the screenshots.